vendor/github.com/intel/govmm/qemu/qmp.go

```diff
@@ -1509,3 +1515,23 @@ func (q *QMP) ExecuteMigrationIncoming(ctx context.Context, uri string) error {
 	}
 	return q.executeCommand(ctx, "migrate-incoming", args, nil)
 }
+
+// ExecuteQueryStatus queries guest status
+func (q *QMP) ExecuteQueryStatus(ctx context.Context) (StatusInfo, error) {
+	response, err := q.executeCommandWithResponse(ctx, "query-status", nil, nil, nil)
+	if err != nil {
+		return StatusInfo{}, err
+	}
+
+	data, err := json.Marshal(response)
+	if err != nil {
+		return StatusInfo{}, fmt.Errorf("unable to extract migrate status information: %v", err)
```

Comment:
@bergwolf Should this be "migrate status"?

Reply:
oops, that one should be fixed in govmm instead.